firebase-test/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Testing Firebase!</title>
+  </head>
+  
+  <body>
+  <h1>Testing Firebase!</h1>
+  <p id="message"></p>
```

Comment:
Looks like your addition of the second paragraph in your HTML file here isn't showing up though -- di dyou remember to commit your changes to this file too?